### DISCUSSION TITLE
billing_code_modifier not required field

### DISCUSSION THREAD

**Discussion by jltang-kp** [CMSgov/price-transparency-guide#475] [Q&A] [answered]
We are having internal discussions about the CPT code modifier field and if we need to publish them in MRF1 since its marked as a not required field. Does any other orgs have opinions on this.

**Comment by rgouldkc** [comment, marked as answer]
Essentially not required makes the field optional. We only use the modifier field if we need to report a rate for the code and modifier that is different from the rate for the code without the modifier.

**Reply by jltang-kp** [reply]
@rgouldkc Thanks, we were thinking the opposite of where if we have a CPT code rate without modifier we would not need to post the rate with modifier.
